Lab3/Lab3/Lab3.cpp

```diff
@@ -0,0 +1,112 @@
+﻿// Lab3.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
+//
+
+#include "pch.h"
+#include <iostream>
+#include <map>
+#include <string>
+#include <iomanip>
+#include <conio.h>
+
+using namespace std;
+
+
+namespace lab3
+{	
+
+	void lab3_2()
+	{
+		printf("Task 2"); 
+		cout << endl;
+
+		float x = 3, y;
+		y = x * x + sin(x);
+
+		printf("y = %f", y);
+
+		_getch();
+		cout << endl;
+	}
+	void lab3_3()
+	{
+		printf("Task 3"); 
+		cout << endl;
+
+		double
+			t, u,
+			k = 4,
+			a = 4.1,
+			x = 5e-5;
+
+		t = 2 * tan(k) / a + log(3 + x) + exp(x);
+		u = sqrt(t + 1) - sin(x) * cos(t);
+
+		printf("t = %f", t); 
+		cout << endl;
+		printf("u = %f", u);
+		cout << endl;
+
+		_getch();
+	}
+	void lab3_4()
+	{
+		printf("Task 4"); 
+		cout << endl;
+		printf("Номер задачи выбрал первый из таблицы"); 
+		cout << endl;
+
+		double
+			t, u,
+			k = 4,
+			x = 2e-4,
+			a = 8.1;
+
+		t = 2 * k / a + log(2 + x);
+		u = sqrt(k - 1) / (t + 1);
+
+		printf("t = %f", t); 
+		cout << endl;
+		printf("u = %f", u);
+		cout << endl;
+
+		_getch();
+
+	}
+	void lab3_5()
```

Comment:
In general this should be written in another way. Your `lab3_*` functions should contain only calculation logic and return value. So all printing and formatting should be in the `main`